tests/CMakeLists.txt

```diff
@@ -88,6 +88,7 @@ if(WITH_PYTHON)
   add_fdb_test(TEST_FILES StreamingWrite.txt IGNORE)
   add_fdb_test(TEST_FILES ThreadSafety.txt IGNORE)
   add_fdb_test(TEST_FILES TraceEventMetrics.txt IGNORE)
+  add_fdb_test(TEST_FILES UnitTestTLogGroup.txt IGNORE)
```

Comment:
The latest tests are specified in `TOML` format. So maybe put the test file in `tests/ptxn/UnitTestTLogGroup.toml`?